css/house-style.md

```diff
@@ -122,33 +122,6 @@ span {
 }
 ```
 
-**Items that semantically MUST sit within other items**. We do this:
```

Comment:
why has this been deleted?

Reply:
Hm, this somehow ended up in this PR, although it comes from [another PR that already has been approved](https://github.com/springernature/frontend-playbook/pull/447).
In that one, I was deleting a section that propoesed something that according to our Playbook we should not do (styling HTML elements rather than classes).

Reply:
Can we remove from this PR please? Does this branch need updating?

Reply:
I was hoping to get around this, but I agree that this is not nice, so I'll update the branch. Thanks for pointing that out.

Reply:
Think this still needs doing? (sorry)

Reply:
Sorry for leaving it untouched that long. I now reverted the change and hope that it will find it's way from the even older PR...

Reply:
No wait, it already has been merged. I now merged the current master into this PR in order to prevent my revert from undoing the change that already landed in master.